pkg/loghttp/push/otlp.go

```diff
@@ -306,6 +310,36 @@ func otlpToLokiPushRequest(ctx context.Context, ld plog.Logs, userID string, ten
 	return pr
 }
 
+func recordScopeUsage(stats *Stats, resAttrs pcommon.Map, scope pcommon.InstrumentationScope, logs int) {
+	var lang string
+	l, ok := resAttrs.Get(attrSdkLanguage)
+	if ok {
+		lang = l.AsString()
```

Comment:
Shouldn't we use an "allow-list" of values rather than using the value provided by the user?

Reply:
what are we protecting against?
A malicious user could just use different sdk verisons.